### PR TITLE
SDI-226: ⚰ Remove dead code

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/OpenApiDocsTest.kt [hunk 1]
```diff
@@ -5,7 +5,7 @@ import net.minidev.json.JSONArray
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.boot.web.server.LocalServerPort
+import org.springframework.boot.test.web.server.LocalServerPort
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/OpenApiDocsTest.kt:8]: fix as was causing deprecated warning in intellij

File: src/test/java/uk/gov/justice/hmpps/prison/service/StaffServiceImplTest.java [hunk 2]
```diff
@@ -94,9 +85,7 @@ public void testNoneExist() throws NoContentException {
         when(staffRepository.findByStaffId(ID_NONE)).thenReturn(getValidStaffDetails(ID_NONE));
         when(staffRepository.findEmailAddressesForStaffId(ID_NONE)).thenReturn(emptyList);
 
-        assertThatThrownBy(() -> {
-            final var addresses = staffService.getStaffEmailAddresses(ID_NONE);
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/hmpps/prison/service/StaffServiceImplTest.java:98]: unused var

File: src/test/java/uk/gov/justice/hmpps/prison/service/StaffServiceImplTest.java [hunk 3]
```diff
@@ -106,9 +95,7 @@ public void testNoneExist() throws NoContentException {
     public void testInvalidId() throws EntityNotFoundException, NoContentException {
         when(staffRepository.findByStaffId(ID_BAD)).thenThrow(EntityNotFoundException.withId(ID_BAD));
 
-        assertThatThrownBy(() -> {
-            final var addresses = staffService.getStaffEmailAddresses(ID_BAD);
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/hmpps/prison/service/StaffServiceImplTest.java:110]: unused var

**Review by Mjwillis — APPROVED** [review]
Lgtm

**Pull request merged**